source/python/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2016 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal.II distribution.
+##
+## ---------------------------------------------------------------------
+
+FIND_LIBRARY(Boost_PYTHON_LIBRARY boost_python PATH ${Boost_LIBRARY_DIRS} NO_DEFAULT_PATH)
+INCLUDE(FindPythonLibs)
+INCLUDE(FindPythonInterp)
+INCLUDE_DIRECTORIES(BEFORE ${CMAKE_CURRENT_BINARY_DIR})
+INCLUDE_DIRECTORIES(SYSTEM ${PYTHON_INCLUDE_DIRS})
+
+SET(_src
+  wrappers.cc
+  export_cell_accessor.cc
+  export_point.cc
+  export_triangulation.cc
+  cell_accessor_wrapper.cc
+  point_wrapper.cc
+  triangulation_wrapper.cc
+  )
+
+FILE(GLOB _header
+  ${CMAKE_SOURCE_DIR}/include/deal.II/python/*.h
+  )
+
+
+PYTHON_ADD_MODULE(PyDealII ${_src})
+
+SET(PYTHON_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX}/lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages)
+
+TARGET_LINK_LIBRARIES(PyDealII deal_II.g)
```

Comment:
Is there a specific reason why we always link against the debug library?

Or more generally, how shall a switch between debug and release library happen on python side?